src/app/modules/users/user.entity.ts

```diff
@@ -26,6 +26,13 @@ export class UserEntity {
   })
     email: string;
 
+  @Column({
```

Comment:
Use another type for this field. Ppostgres has specialized types for IPs. https://www.sqlshack.com/an-introduction-to-postgresql-data-types-with-examples/#:~:text=PostgreSQL%20data%20types%20provide%20data,or%2019%2Dbyte%20storage%20size.